scripts/rki-update.py

```diff
@@ -19,38 +19,35 @@
     print("You can install them with:  pip install pandas dfply")
     sys.exit(1)
 
-if len(sys.argv) != 2:
+if len(sys.argv) != 3:
     print("Usage:  python rki-update.py [RKI-filename.csv]")
     sys.exit(1)
 
-print("Reading csv:", sys.argv[1])
-
-csv = pd.read_csv(sys.argv[1], parse_dates=True)
-
-# split out dates
-csv >>= separate(
-    X.NeuGenesen,  # This is actually the X.Refdatum but unix "cut" command is off-by-one
-    ["year", "dash1", "month", "dash2", "day"],
-    sep=[4, 5, 7, 8, 10],
-    convert=True,
-    remove=False,
-) >> separate(
-    X.Meldedatum,
-    ["myear", "dash1", "mmonth", "dash2", "mday"],
-    sep=[4, 5, 7, 8, 10],
-    convert=True,
-    remove=False,
-) >> select(X.Bundesland, X.Landkreis, X.AnzahlFall, X.year, X.month, X.day, X.myear, X.mmonth, X.mday)
+print("Reading csv bundeslaender:", sys.argv[1])
+csv_bundeslaender = pd.read_csv(sys.argv[1], parse_dates=True)
+
+print("Reading csv landkreise:", sys.argv[2])
+csv_landkreise = pd.read_csv(sys.argv[2], parse_dates=True)
+
+# Split the Meldedatum into year, month and date
+csv_bundeslaender[['year','month', 'day']] = csv_bundeslaender['Meldedatum'].str.split('-',expand=True)
+
+# Remove unused Columns
+csv_bundeslaender = csv_bundeslaender.drop(csv_bundeslaender.columns[[2, 3, 4, 6, 7]],axis = 1)
+
+# Split the Meldedatum into year, month and date
+csv_landkreise[['year','month', 'day']] = csv_landkreise['Meldedatum'].str.split('-',expand=True)
+# Remove unused Columns
+csv_landkreise = csv_landkreise.drop(csv_landkreise.columns[[2, 3, 5, 6]],axis = 1)
 
 # Berlin
 fname = "berlin-cases.csv"
 print(fname)
 berlin = (
-    csv
-    >> filter_by(X.Bundesland == "Berlin")
-    >> group_by(X.Bundesland, X.year, X.month, X.day)
-    >> summarize(cases=X.AnzahlFall.sum())
-    >> arrange(X.year, X.month, X.day)
+    csv_bundeslaender
+    >> filter_by(X.Bundesland_id == 11)
+    >> group_by(X.Meldedatum, X.year, X.month, X.day)
```

Comment:
This groups by Meldedatum AND year/month/day? Is this necessary? Previously we were grouping by Bundesland and by year/month/day